tests/foreman/cli/test_computeresource_rhev.py

```diff
@@ -0,0 +1,353 @@
+# -*- encoding: utf-8 -*-
+"""
+:Requirement: Computeresource
+
+:CaseLevel: Acceptance
+
+:CaseComponent: CLI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from robottelo.decorators import (
+    run_only_on,
+    stubbed,
+    skip_if_bug_open,
+    tier1,
+    tier2,
+    tier3
+)
+from robottelo.test import CLITestCase
+
+
+class RHEVComputeResourceTestCase(CLITestCase):
+    """RHEVComputeResource CLI tests."""
+
+    @tier1
+    @run_only_on('sat')
+    @stubbed()
+    def test_positive_create_rhev_with_valid_name(self):
+        """Create Compute Resource of type Rhev with valid name
+
+        :id: 92a577db-144e-4761-a52e-e83887464986
+
+        :steps: Create a Compute resource of type rhev with valid name.
+
+        :expectedresults: Compute resource is created
+
+        :caseautomation: notautomated
+
+        :CaseImportance: Critical
+        """
+
+    @tier1
+    @run_only_on('sat')
+    @stubbed()
+    def test_positive_rhev_info(self):
+        """List the info of RHEV compute resource
+
+        :id: 1b18f6e8-c431-41ab-ae49-a2bbb74712f2
+
+        :steps: Display the information of RHEV compute resource.
+
+        :expectedresults: RHEV Compute resource Info is displayed
+
+        :caseautomation: notautomated
+
+        :CaseImportance: Critical
+        """
+
+    @tier1
+    @run_only_on('sat')
+    @stubbed()
+    def test_positive_delete_by_name(self):
+        """Delete the RHEV compute resource by name
+
+        :id: ac84acbe-3e02-4f49-9695-b668df28b353
+
+        :steps: Delete the RHEV compute resource with name parameter
+
+        :expectedresults: Compute resource is deleted
+
+        :caseautomation: notautomated
+
+        :CaseImportance: Critical
+        """
+
+    @tier1
+    @run_only_on('sat')
+    @stubbed()
+    def test_positive_delete_by_id(self):
+        """Delete the RHEV compute resource by id
+
+        :id: 4bcd4fa3-df8b-4773-b142-e47458116552
+
+        :steps: Delete the RHEV compute resource with name id
+
+        :expectedresults: Compute resource is deleted
+
+        :caseautomation: notautomated
+
+        :CaseImportance: Critical
+        """
+
+    @tier1
+    @run_only_on('sat')
+    @stubbed()
+    def test_negative_create_rhev_with_url(self):
+        """RHEV compute resource negative create with invalid values
+
+        :id: 1f318a4b-8dca-491b-b56d-cff773ed624e
+
+        :steps: Create RHEV compute resource with invalid url for url parameter
```

Comment:
same here and in all test cases that has one step that sounds the same with the description